network-api/networkapi/templates/fragments/buyersguide/article_card_horizontal.html

```diff
@@ -1,12 +1,10 @@
 {% load localization wagtailimages_tags %}
 
 <div class="tw-flex tw-flex-row tw-gap-3 medium:tw-gap-5 tw-w-full">
-  <div class="medium:tw-max-w-xs large:tw-max-w-sm medium:tw-shrink-0">
-    <a href="{% relocalized_url page.localized.url %}">
-      {% image page.localized.get_meta_image original class="tw-hidden medium:tw-block tw-max-h-[131px] tw-max-w-[255px] tw-object-cover" %}
-      {% image page.localized.get_meta_image fill-100x100 class="tw-block medium:tw-hidden tw-min-h-[78px] tw-min-w-[78px] tw-rounded-2xl tw-object-cover" %}
-    </a>
-  </div>
+  <a href="{% relocalized_url page.localized.url %}" class="tw-block tw-shrink-0 tw-max-w-[256px]">
+    {% image page.localized.get_meta_image fill-512x288 class="tw-hidden medium:tw-block tw-w-[256px] tw-aspect-video tw-object-cover" %}
+    {% image page.localized.get_meta_image fill-160x160 class="tw-block medium:tw-hidden tw-w-[80px] tw-aspect-square tw-rounded-2xl tw-object-cover" %}
+  </a>
```

Comment:
**Question - non blocking**

Just checking if this is missing the title value as per https://github.com/mozilla/foundation.mozilla.org/pull/9413/files#diff-bc84a2748cfdc09d3d023bc2b4ac8f4c5b2c7eada332d5beb087980cc3da2e66R11

Reply:
The title is displayed in the card body and we are passing the page through to that include.